packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1224,17 +1212,19 @@ export default class NewVm extends BaseComponent {
     } = this.state.state
     switch (installMethod) {
       case 'customConfig':
-        return customConfig || !configDrive
+        return customConfig || installMethod === 'noConfigDrive'
```

Comment:
It should be good even if `customConfig === ''` since you're falling back to `DEFAULT_CLOUD_CONFIG_TEMPLATE`.